.github/CODEOWNERS

```
@@ -1,4 +1,4 @@
 # Lines starting with '#' are comments.
 # Each line is a file pattern followed by one or more owners.
 
-*                                    @MRabenda @davidwgrossman @MetaMask/devs
+*                                    @MRabenda @davidwgrossman @MetaMask/engineering
```

Looks like this other username no longer exists. We should remove that as well to resolve any CODEOWNER file errors:
```suggestion
*                                    @MRabenda @MetaMask/engineering
```